std/bigint.d

```diff
@@ -61,7 +68,6 @@ public:
             s = s[1..$];
         }
         data = 0UL;
-        auto q = 0X3;
```

Comment:
This appears to be dead code leftover from a very long time ago, when BigInt was first merged into Phobos.